File: Clase6/ejemploC#Java/persona.java
import java.io.Serializable;

public class persona implements Serializable{
	private String nombre;
	private int edad;
	
	public void setNombre(String nombre){
		this.nombre = nombre;
	}
	
	public String getNombre(){
		return nombre;
	}
	
	public void setEdad(int edad){
		this.edad = edad;
	}
	
	public int getEdad(){
		return edad;
	}
}

File: Clase6/ejemploC#Java/persona.cs
using System;
[Serializable]
public class persona{
	private string nombre;
	private int edad;
	
	public string Nombre{
		set{
			nombre = value;
		}
		get{
			return nombre;
		}
	}
	
	public int Edad{
		set{
			edad = value;
		}
		get{
			return edad;
		}
	}
}